### PR TITLE
OP-1064: Switch buttons always visible, make user roles unique

### PR DESCRIPTION
**Pull request by olewandowski1**
[OP-1064](https://openimis.atlassian.net/browse/OP-1064)

In scope of this ticket, I dealt with the disapearing switch buttons. I made them always visible as we settled in the daily. Moreover, I made a user roles unique. This prevent adding the same role multiple times. Now, I can edit the user and manipulate his roles by using switchs as well as input. It makes the whole form more user-friendly.

[OP-1064]: https://openimis.atlassian.net/browse/OP-1064?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/pickers/UserRolesPicker.js [hunk 1]
```diff
@@ -32,6 +32,9 @@ const UserRolesPicker = ({
     { str: searchString },
   );
 
+  const roles = data?.role?.edges.map((edge) => edge.node) ?? [];
+  const uniqueValues = [...new Map(value?.map((role) => [role.isSystem, role])).values()];
```
Comment (dborowiecki) [inline src/components/pickers/UserRolesPicker.js:36]: `isSystem` value determines that given role is built into openIMIS system (schema admin, CA, etc.). Administrators can add custom roles that have `isSystem` set to null. Witch this check, custom roles would be treated as duplicates. Can we change this to other property (e.g. id)?

**Review by dborowiecki — CHANGES_REQUESTED** [review]

File: src/components/pickers/UserRolesPicker.js (hunk 1 above)
Comment (olewandowski1) [reply src/components/pickers/UserRolesPicker.js]: Yes, you are right. I changed `isSystem` to the `id` property.

**Review by dborowiecki — APPROVED** [review]
LGTM

File: src/components/pickers/UserRolesPicker.js (hunk 1 above)
Comment (dborowiecki) [reply src/components/pickers/UserRolesPicker.js]: Thanks

**Pull request merged**